.idea/.name

```diff
@@ -0,0 +1 @@
+intl-tel-input
```

Comment:
Add the `.idea` directory to `.gitignore`, since those are settings for your environment and not source code for intl-tel-input.